rlclientlib/serialization/fb_serializer.h

```diff
@@ -269,32 +269,45 @@ namespace reinforcement_learning { namespace logger {
     flatbuffer_allocator _allocator;
     flatbuffers::FlatBufferBuilder _builder;
     buffer_t& _buffer;
-    content_encoding_enum _content_encoding;
+    const char* _content_encoding;
     flatbuffers::Offset<v2::BatchMetadata> _batch_metadata_offset;
   };
 
   template <>
   struct fb_event_serializer<generic_event> {
-    using fb_event_t = v2::Event;
+    using fb_event_t = v2::SerializedEvent;
     using offset_vector_t = typename std::vector<flatbuffers::Offset<fb_event_t>>;
     using batch_builder_t = v2::EventBatchBuilder;
 
     static size_t size_estimate(const generic_event& evt) {
       return evt.get_payload().size();
     }
 
-    static int serialize(generic_event& evt, flatbuffers::FlatBufferBuilder& builder,
+    static int serialize(generic_event& evt, flatbuffers::FlatBufferBuilder& outter_builder,
       flatbuffers::Offset<fb_event_t>& ret_val, api_status* status) {
 
-      const auto id_offset = builder.CreateString(evt.get_id());
+      flatbuffers::FlatBufferBuilder builder;
 
       const auto& ts = evt.get_client_time_gmt();
       v2::TimeStamp client_ts(ts.year, ts.month, ts.day, ts.hour,
         ts.minute, ts.second, ts.sub_second);
-      const auto meta_offset = v2::CreateMetadataDirect(builder, evt.get_id(), &client_ts, nullptr, evt.get_payload_type(), evt.get_pass_prob());
+      const auto meta_offset = v2::CreateMetadataDirect(
+        builder, 
+        evt.get_id(), 
+        &client_ts, 
+        nullptr, //FIXME app-id
```

Comment:
Does this need to be fixed before the PR is merged?

Reply:
after, this was already there before this PR.